addons/website_slides/static/src/tests/activity_tests.js

```diff
@@ -0,0 +1,121 @@
+odoo.define('website_slides/static/src/tests/activity_tests.js', function (require) {
+'use strict';
+
+const components = {
+    Activity: require('mail/static/src/components/activity/activity.js'),
+};
+
+const {
+    afterEach,
+    beforeEach,
+    start,
+} = require('mail/static/src/utils/test_utils.js');
+
+QUnit.module('website_slides', {}, function () {
+QUnit.module('activity_tests.js', {
```

Comment:
It sucks we don't have bugfix files for `website_livechat`, which is why we have to pollute other files in 14.0. Will improve that in master.

For the test, since introducing new file is acceptable, I would make it as if this is all files where created as expected in master. This means this test should be in `website_slides/static/src/components/activity/activity_tests.js`.

QUnit modules should follow file path:
```js
QUnit.module('website_slides', {}, function () {
QUnit.module('components', {}, function () {
QUnit.module('activity', {}, function () {
QUnit.module('activity.js', {}, function () {
```